packages/react-components/react-infobutton/src/components/InfoButton/InfoButton.types.ts

```diff
@@ -1,15 +1,29 @@
+import type { Button } from '@fluentui/react-button';
+import type { PopoverProps, PopoverSurface } from '@fluentui/react-popover';
 import type { ComponentProps, ComponentState, Slot } from '@fluentui/react-utilities';
 
 export type InfoButtonSlots = {
-  root: Slot<'div'>;
+  root: NonNullable<Slot<PopoverProps>>;
+
+  /**
+   * The button that triggers the Popover.
+   */
+  trigger: NonNullable<Slot<typeof Button>>;
+
+  /**
+   * The content to be displayed in the Popover.
+   */
+  content: NonNullable<Slot<typeof PopoverSurface>>;
 };
 
 /**
  * InfoButton Props
  */
-export type InfoButtonProps = ComponentProps<InfoButtonSlots> & {};
+export type InfoButtonProps = Omit<ComponentProps<Partial<InfoButtonSlots>>, 'children'>;
```

Comment:
The design spec mentions allowing `subtle` and `transparent` button appearances, but this will be done in the slots. This comes with a small problem; we aren't restricting the other appearances. Once the spec is merged, this will be decided.

It also mentions providing a way to change the position of the popover, which will also be provided by the popover (root) slot.